includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -570,14 +569,18 @@ public function add_smooth_scrolling( $link_xpaths ) {
 	 * @link https://caniuse.com/#feat=svg
 	 */
 	public function force_svg_support() {
-		$this->dom->documentElement->setAttribute(
-			'class',
-			preg_replace(
-				'/(^|\s)no-svg(\s|$)/',
-				' svg ',
-				$this->dom->documentElement->getAttribute( 'class' )
-			)
-		);
+		$class = $this->dom->documentElement->getAttribute( 'class' );
+
+		if ( strpos( $class, 'no-svg' ) ) {
```

Comment:
This will fail with `class="no-svg"` since the position will be `0`.

But why the need for wrapping the condition? If the string isn't found then just no change will be made.

Reply:
> This will fail with class="no-svg" since the position will be 0.
Made the fix just as you commented :dash:.

> But why the need for wrapping the condition?
An empty class attribute (`class=""`) will be set if there was no class attribute.